java/code/src/com/redhat/rhn/manager/access/AccessGroupManager.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2025 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ */
+
+package com.redhat.rhn.manager.access;
+
+import static com.redhat.rhn.domain.role.RoleFactory.ORG_ADMIN;
+import static com.redhat.rhn.domain.role.RoleFactory.SAT_ADMIN;
+
+import com.redhat.rhn.domain.access.AccessGroup;
+import com.redhat.rhn.domain.access.AccessGroupFactory;
+import com.redhat.rhn.domain.access.Namespace;
+import com.redhat.rhn.domain.access.NamespaceFactory;
+import com.redhat.rhn.domain.org.Org;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Class to manage RBAC access groups
+ */
+public class AccessGroupManager {
+
+    private static final Logger LOG = LogManager.getLogger(AccessGroupManager.class);
```

Comment:
The LOG variable is never used. It would be good to add several debug data in case we need to activate it and check what is happening.